File: src/main/java/com/atsuishio/superbwarfare/compat/coldsweat/ColdSweatCompatHandler.java
package com.atsuishio.superbwarfare.compat.coldsweat;

public class ColdSweatCompatHandler {

    /*
    public static void onPlayerInVehicle(TickEvent.PlayerTickEvent event) {
        var player = event.player;
        if (player == null) return;
        if (player.getVehicle() instanceof VehicleEntity vehicle
                && vehicle.hasEnergyStorage()
                && vehicle.isEnclosed(vehicle.getSeatIndex(player))
                && vehicle.getEnergy() > 0
        ) {
            Temperature.set(player, Temperature.Trait.CORE, 1);
        }
    }

    public static boolean hasMod() {
        return ModList.get().isLoaded(CompatHolder.COLD_SWEAT);
    }
    */
}

File: src/main/kotlin/com/atsuishio/superbwarfare/compat/coldsweat/ColdSweatCompatHandler.kt
package com.atsuishio.superbwarfare.compat.coldsweat

/*
object ColdSweatCompatHandler {
    @JvmStatic
    fun onPlayerInVehicle(event: PlayerTickEvent) {
        val player = event.player ?: return
        val vehicle = player.vehicle
        if (vehicle is VehicleEntity
            && vehicle.hasEnergyStorage()
            && vehicle.isEnclosed(vehicle.getSeatIndex(player))
            && vehicle.energy > 0
        ) {
            Temperature.set(player, Temperature.Trait.CORE, 1.0)
        }
    }

    fun hasMod(): Boolean {
        return ModList.get().isLoaded(CompatHolder.COLD_SWEAT)
    }
}
*/
